styles/main.css

```diff
@@ -1,283 +1,314 @@
-*{
+* {
   text-decoration: none;
-  font-family: Lato, serif;
-  font-display: swap;
+  font-family: "Lato", sans-serif;
 }
 *,
 *:after,
-*:before{
+*:before {
   box-sizing: border-box;
 }
-ul{
+ul {
   padding-left: 0;
 }
-body{
-  background-color: #bebbbb;
-}
-.container{
+.container {
   max-width: 1140px;
   margin-left: auto;
   margin-right: auto;
   padding-left: 15px;
   padding-right: 15px;
 }
-.row{
+.row {
   margin-left: -15px;
   margin-right: -15px;
 }
-.row:after{
+.row:after {
   content: '';
   display: block;
   clear: both;
 }
-.col{
+.col {
   padding-left: 15px;
   padding-right: 15px;
   float: left;
 }
-.col-1{
+.col-1 {
   float: left;
```

Comment:
We need float property only for .col class. remove, please. Fix all the col- classes.